configs/grumphp-extension.yml

```diff
@@ -37,75 +38,84 @@ grumphp:
         - "\\$_[A-Z_]+\\["
       regexp_type: E
       triggered_by:
-        - php
         - inc
         - install
         - module
-        - theme
+        - php
         - profile
+        - theme
     git_branch_name:
       whitelist:
-        - "#^\\d+\\(.\\d+)?\\.x|feature/([A-Z][A-Z0-9]+-\\d+|[a-z][a-z0-9]*(-[a-z0-9]+)*)$#"
+        - "#^\\d+(.\\d+)?\\.x|master|develop|feature/([A-Z][A-Z0-9]+-\\d+|[a-z][a-z0-9]*(-[a-z0-9]+)*)$#"
```

Comment:
`master` en `develop` staan hier bewust niet tussen, we gebruiken normaal enkel versie specifieke branches.

De `\\` die verwijderd zijn hoorden voor het puntje (was fout in de upstream branch).

Reply:
Updated.